client/settings/fraud-protection/index.js

```diff
@@ -0,0 +1,501 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf, __ } from '@wordpress/i18n';
+import { Button, Card, Modal, Notice } from '@wordpress/components';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import CardBody from '../card-body';
+import { useCurrentProtectionLevel, useCurrencies } from '../../data';
+import TipIcon from 'wcpay/icons';
+import './style.scss';
+
+const ExceedsDollarAmountRuleUSDStore = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<li>
+				{ __( 'An order exceeds ', 'woocommerce-payments' ) }{ ' ' }
+				<strong>{ __( '$1,000.00.', 'woocommerce-payments' ) }</strong>
+			</li>
+		);
+	}
+
+	return (
+		<li>
+			{ __( 'An order exceeds', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '$1,000.00', 'woocommerce-payments' ) } </strong>
+			{ __( 'or', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '10 items.', 'woocommerce-payments' ) }</strong>
+		</li>
+	);
+};
+
+const ExceedsDollarAmountRuleNotUSDStore = ( props ) => {
+	if ( 'high' === props.level ) {
+		return (
+			<li>
+				{ __(
+					'An order exceeds the equivalent of',
+					'woocommerce-payments'
+				) }{ ' ' }
+				<strong>
+					{ __( '$1,000.00 USD', 'woocommerce-payments' ) }
+				</strong>{ ' ' }
+				{ sprintf(
+					__( 'in %s.', 'woocommerce-payments' ),
+					props.defaultCurrency
+				) }
+			</li>
+		);
+	}
+
+	return (
+		<li>
+			{ __(
+				'An order exceeds the equivalent of',
+				'woocommerce-payments'
+			) }{ ' ' }
+			<strong>{ __( '$1,000.00 USD', 'woocommerce-payments' ) } </strong>{ ' ' }
+			{ sprintf(
+				__( 'in %s', 'woocommerce-payments' ),
+				props.defaultCurrency
+			) }{ ' ' }
+			{ __( 'or', 'woocommerce-payments' ) }{ ' ' }
+			<strong>{ __( '10 items.', 'woocommerce-payments' ) }</strong>
+		</li>
+	);
+};
+
+const ProtectionLevelModalNotice = ( props ) => {
```

Comment:
Similarly to the previous one, we can merge the component structure and use ternary to determine the notice text.

```js
const ProtectionLevelModalNotice = ( { level } ) => {
	const isHighProtectionLevel = 'high' === level;

	return (
		<Notice
			className="component-notice--is-info"
			status="info"
			isDismissible={ false }
		>
			<div className="component-notice__content--flex">
				<TipIcon className="component-notice__icon" />
				<p>
					{ isHighProtectionLevel
						? __(
								'Offers the highest level of filtering for stores, but may catch some legitimate transactions',
								'woocommerce-payments'
						  )
						: __(
								"Provides a standard level of filtering that's suitable for most business.",
								'woocommerce-payments'
						  ) }
				</p>
			</div>
		</Notice>
	);
};
```

Reply:
Thanks for pointing this out. Fixed in [0835f2c](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/0835f2cdda206552538b6a5acb66bdd083a8a83a).